kimchi/src/curve.rs

```diff
@@ -78,13 +79,13 @@ impl KimchiCurve for GroupAffine<VestaParameters> {
     }
 
     fn other_curve_prime_subgroup_generator() -> (Self::ScalarField, Self::ScalarField) {
-        GroupAffine::<PallasParameters>::prime_subgroup_generator()
+        Affine::<PallasParameters>::generator()
```

Comment:
It seems we do not need to verify it is in the prime subgroup. But reviewers should check twice.